packages/grafana-data/src/utils/dataLinks.ts

```diff
@@ -62,6 +63,7 @@ export function mapInternalLinkToExplore(
           onClickFn?.({
             datasourceUid: link.internal!.datasourceUid,
             query: interpolatedQuery,
+            range,
```

Comment:
This sort of affects the links from logs in loki but as this range is from the left split, which is what would have been used for the right anyway, this should not change the behaviour of log links.